quixstreams/state/manager.py

```diff
@@ -19,6 +19,7 @@
 
```

Comment:
In this file, you've got to update the docstring of `stores` property to say `{stream_id: {store_name: store}}`